### DISCUSSION TITLE
GitHub CI with brew test-bot does not work, "sqlite: no bottle available!"

### DISCUSSION THREAD

**Discussion by slhck** [Homebrew/discussions#1233] [Tap maintenance and Homebrew development] [answered]
See the [failed run](https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/pull/78/checks?check_run_id=2307627036) here.

All we do is run:

```
brew test-bot --skip-recursive-dependents Formula/ffmpeg.rb
```

It reproducibly errors with:

```
Error: sqlite: no bottle available!
You can try to install from source with:
  brew install --build-from-source sqlite
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

What gives? Shouldn't there be a bottle for sqlite?

Curiously, this [previous run](https://github.com/homebrew-ffmpeg/homebrew-ffmpeg/runs/2304480463?check_suite_focus=true) worked fine.

**Comment by iMichka** [comment, marked as answer]
We have issues with our CI setup, I'm trying to fix this.

**Reply by slhck** [reply]
As far as I can tell, it seems to work again.
